pghoard/config.py

```diff
@@ -23,16 +22,57 @@ def get_cpu_count():
     return multiprocessing.cpu_count()
 
 
-def find_pg_binary(program, versions=None):
+def get_command_version(command):
```

Comment:
I would add type hints, especially since we hide failure by returning `None` instead.